domains/glauroqj.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "glauroqj personal site",
+  "owner": {
+    "username": "glauroqj",
+    "email": "glauro.juliani@hotmail.com"
+  },
+  "repo": "https://github.com/glauroqj/register",
+  "record": {
+    "URL": "http://glauroqj.is-a.dev"
```

Comment:
This is an invalid record file. If you have a github pages repository, please a cname record as `glauroqj.github.io`